Task2_2_MatrixClass.cs

```diff
@@ -0,0 +1,101 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task2_2
+{
+    public class MatrixElementsDiagonal
+    {
+        private int[] diagonalElements;
+
+        public int Size { get; }
+
+        public MatrixElementsDiagonal(params int[] elements)
+        {
+            if (elements == null)
+            {
+                Size = 0;
+                diagonalElements = new int[0];
+            }
+            else
+            {
+                Size = elements.Length;
+                diagonalElements = new int[Size];
+                for (int i = 0; i < Size; i++)
+                {
+                    diagonalElements[i] = elements[i];
+                }
+            }
+        }
+
+        public int this[int i, int j]
+        {
```

Comment:
Where is set part for indexer? )) It's necessary

Reply:
set added